### PR TITLE
Devellop

### PR DESCRIPTION
**Pull request by benjivou**
Fix the last pull request

### CODE REVIEW

File: app/src/main/java/com/example/movieapp/crawler/pojo/Movie.kt [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package com.example.movieapp.crawler.pojo
+
+import com.google.gson.annotations.SerializedName
+import java.util.*
+
+/**
+ * Created by Benjamin Vouillon on 08,July,2020
+ */
+data class Movie(
+    val popularity: Float,
```
Comment (antoine-purnelle-lb) [inline app/src/main/java/com/example/movieapp/crawler/pojo/Movie.kt:10]: Put the @SerializedName annotation on each field even if the name doesn't change. 
It'll prevent an issue where you would rename the val and break your code. 
Also, put a space after the parenthesis or an linebreak ;-)

File: app/src/main/java/com/example/movieapp/crawler/pojo/ServerRequestContent.kt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.example.movieapp.crawler.pojo
+
+import com.google.gson.annotations.SerializedName
+
+/**
+ * Created by Benjamin Vouillon on 08,July,2020
+ */
+data class ServerRequestContent(
+    val page: Int,
```
Comment (antoine-purnelle-lb) [inline app/src/main/java/com/example/movieapp/crawler/pojo/ServerRequestContent.kt:9]: Same here :-)

File: app/src/main/java/com/example/movieapp/viewModel/MainViewModel.kt [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    RATED,
+    LIKED
+}
+
+class MainViewModel : ViewModel() {
+
+    /**
+     * Our 3 lists of Movies
+     */
+    var popular:MutableLiveData<List<Movie>> = internetCall(TypeList.POPULAR)
```
Comment (antoine-purnelle-lb) [inline app/src/main/java/com/example/movieapp/viewModel/MainViewModel.kt:26]: What @alba-benoit said ;-)

**Review by antoine-purnelle-lb — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/movieapp/crawler/pojo/Movie.kt (hunk 1 above)
Comment (benjivou) [reply app/src/main/java/com/example/movieapp/crawler/pojo/Movie.kt:10]: @SerializedName done
add spaces

File: app/src/main/java/com/example/movieapp/crawler/pojo/ServerRequestContent.kt (hunk 2 above)
Comment (benjivou) [reply app/src/main/java/com/example/movieapp/crawler/pojo/ServerRequestContent.kt:9]: Done

File: app/src/main/java/com/example/movieapp/viewModel/MainViewModel.kt (hunk 3 above)
Comment (benjivou) [reply app/src/main/java/com/example/movieapp/viewModel/MainViewModel.kt:26]: Solved

**Pull request closed without merging**